languages/rust/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Tuples
+
+Tuples are a lightweight way to group a bounded set of heterogenous data together. A tuple doesn't have
```

Comment:
"bounded set of heterogenous data together"

I have to think quote hard (and google) to make sense of this. I don't know what "bounded" means in this context, or "heterogenous" in general.

Would I be correct in saying "Tuples are a lightweight way to group a fixed set of different data together."

Reply:
- "bounded" -> "fixed": sure, I'll agree that wording is clearer
- "heterogeneous" -> "different": the point I was trying to make is the data can be of arbitrary types. "Different" makes it sound like it can be different values, but not necessarily different types; a student might not know what distinguishes a tuple from a fixed-size array.

Reply:
How about then:

> Tuples are a lightweight way to group a fixed set of data together. The data can be made up of different types and the same type with different values.

Reply:
What do you think of the wording as of bbafaaf: 

> Tuples are a lightweight way to group a fixed set of arbitrary types of data together.

Reply:
I agree that heterogeneous might be confusing to people. The [Swift tuples document](https://github.com/exercism/v3/blob/master/languages/swift/concepts/tuples/about.md) describes it as follows:

> Tuples are used to combine multiple values into a single compound value where each of the values may have a different type.

I'm not sure about the word "compound," but the rest of the sentence is very clear I feel.

Reply:
Oh, I missed iHiD's comment. Whoops!

Reply:
> Tuples are a lightweight way to group a fixed set of arbitrary types of data together.

I like it!

Reply:
I like this definition of tuples adapted from _F# 4.0 Design Patterns_, but I think Jeremy  may have written it better:

A tuple is an immutable data type that represents a parenthesis enclosed, comma separated, ordered grouping of arbitrary values. Types of these values are completely arbitrary, and it does not matter whether they are the same or not.